analyzers/analyzer.go

```diff
@@ -3,6 +3,7 @@ package analyzers
 
 import (
 	"errors"
+	"github.com/fossas/fossa-cli/analyzers/haskell"
```

Comment:
if your IDE has an imports grouping feature, I'd suggest turning it on. Otherwise you may want to use [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports) instead. Helps reduce diff churn. E.g., https://github.com/fossas/fossa-cli/blob/master/api/fossa/normalize.go#L3

Reply:
Is it possible for us to add an auto-formatting tool that does this for us?